static/schemas/creative-formats-v1.json

```diff
@@ -0,0 +1,263 @@
+      "video_vertical_mobile": {
+        "format_id": "video_vertical_mobile",
+        "name": "Vertical Mobile Video",
+        "type": "video",
+        "description": "Full-screen vertical video for mobile",
+        "specs": {
+          "resolution": "1080x1920 (9:16)",
+          "duration": "6s, 15s, or 30s",
+          "file_format": "MP4",
+          "features": ["skippable after 5s", "sound off by default"]
+        },
+        "delivery_options": {
+          "hosted": {
+            "supported": true
+          },
+          "vast": {
+            "supported": true,
+            "versions": ["4.0+"],
+            "required_extensions": ["OMID for viewability"]
+          }
+        }
+      },
+      "video_ctv_standard": {
+        "format_id": "video_ctv_standard",
+        "name": "Connected TV Video",
+        "type": "video",
+        "description": "Standard CTV/OTT video ad",
+        "specs": {
+          "resolution": "1920x1080 or 3840x2160 (4K)",
+          "duration": "15s, 30s, or 60s",
+          "file_format": "MP4",
+          "features": ["non-skippable", "sound on by default"]
+        },
+        "delivery_options": {
+          "vast": {
+            "supported": true,
+            "versions": ["4.0+"],
+            "required": true,
+            "features": ["ad pods", "server-side ad insertion"]
+          }
+        }
+      }
+    },
+    "audio": {
+      "audio_streaming": {
+        "format_id": "audio_streaming",
+        "name": "Streaming Audio Ad",
+        "type": "audio",
+        "description": "Audio ad for music/podcast streaming",
+        "specs": {
+          "duration": "15s, 30s, or 60s",
+          "file_format": "MP3 or M4A",
+          "bitrate": "128kbps minimum",
+          "companion_banner": "640x640 optional"
+        },
+        "delivery_options": {
+          "hosted": {
+            "supported": true,
+            "description": "Direct URL to audio file"
+          },
+          "vast": {
+            "supported": true,
+            "versions": ["4.1+"],
+            "notes": "Audio delivered via VAST MediaFile"
+          },
+          "daast": {
+            "supported": false,
+            "reason": "Deprecated in favor of VAST 4.1"
+          }
+        }
+      },
+      "audio_podcast": {
+        "format_id": "audio_podcast",
+        "name": "Podcast Ad",
+        "type": "audio",
+        "description": "Dynamic ad insertion for podcasts",
+        "specs": {
+          "duration": "15s, 30s, 60s, or host-read",
+          "file_format": "MP3",
+          "bitrate": "128kbps minimum",
+          "insertion_type": ["pre-roll", "mid-roll", "post-roll"]
+        },
+        "delivery_options": {
+          "hosted": {
+            "supported": true,
+            "required": true
+          },
+          "vast": {
+            "supported": true,
+            "versions": ["4.1+"]
+          }
+        }
+      }
+    },
+    "display": {
+      "display_banner_728x90": {
```

Comment:
in general I think most standard units should be pretty flexible and generally underspecified. So `display_3p_300x250` should be pretty much just "give me an ad tag" or `display_image_300x250` should be "give me an image, a click url, and optionally some ad verification code". same for other formats.